WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static async Task<Block> DownloadBlockAsync(this Node node, uint256 hash,
 		}
 
 		using var listener = node.CreateListener();
-		var getData = new GetDataPayload(new InventoryVector(node.AddSupportedOptions(InventoryType.MSG_WITNESS_BLOCK), hash));
+		var getData = new GetDataPayload(new InventoryVector(node.AddSupportedOptions(InventoryType.MSG_BLOCK), hash));
```

Comment:
ÀddSupportedOption` adds the WITNESS_FLAG to the message if the node supported witness relay.

Reply:
If this is true and we will want https://github.com/zkSNACKs/WalletWasabi/pull/11139 then that'd be a change compared to the original code, so we may even save bandwidth compared to release?